documentation/processing.tex

```diff
@@ -52,13 +54,13 @@ \subsection{The main loop of manager}
 
 \begin{enumerate}
 	\item When a processor finishes its work, the next processor is selected from the array.
-	\item When a provider finishes --- before the next provider is selected from the array --- manager checks whether it triggers the termination of the current processor as well (see \emph{terminal condition} in \cref{lab06:sect_proc}). If true, perform rule 1.
+	\item When a provider finishes --- before the next provider is selected from the array --- manager checks whether it triggers the termination of the current processor as well (see \emph{terminal condition} in \cref{lab06:sect_proc}). Only if true, perform rule 1.
```

Comment:
Stale z toho podla mna nie je jasne co sa stane ked je to false.
```suggestion
	\item When a provider finishes --- before the next provider is selected from the array --- manager checks whether it triggers the termination of the current processor as well (see \emph{terminal condition} in \cref{lab06:sect_proc}). Only if true, perform rule 1, otherwise the current processor stays active.
```